trantor/net/EventLoop.h

```diff
@@ -121,8 +121,30 @@ class TRANTOR_EXPORT EventLoop : NonCopyable
      * @note If the current thread is the thread of the event loop, the function
      * f is executed directly before the method exiting.
      */
-    void runInLoop(const Func &f);
-    void runInLoop(Func &&f);
+    template <typename Functor>
+    inline void runInLoop(const Functor &f)
+    {
+        if (isInLoopThread())
+        {
+            f();
+        }
+        else
+        {
+            queueInLoop(f);
+        }
+    }
```

Comment:
This is not needed because the following universal template can handle this case correctly.

Reply:
Right. I forgot Functor&& is not a r-valule template reference,